Tree/PhpcrOdmTree.php

```diff
@@ -1,470 +0,0 @@
-<?php
-
-/*
- * This file is part of the Sonata package.
- *
- * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Sonata\DoctrinePHPCRAdminBundle\Tree;
-
-use Doctrine\ODM\PHPCR\Document\Generic;
-use PHPCR\Util\NodeHelper;
-
-use PHPCR\Util\PathHelper;
-use Symfony\Component\PropertyAccess\PropertyAccess;
-use Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException;
-use Symfony\Component\Translation\TranslatorInterface;
-use Symfony\Component\Templating\Helper\CoreAssetsHelper;
-use Symfony\Cmf\Bundle\TreeBrowserBundle\Tree\TreeInterface;
-
-use Doctrine\ODM\PHPCR\DocumentManager;
-use Doctrine\Common\Util\ClassUtils;
-
-use Sonata\AdminBundle\Admin\Pool;
-use Sonata\AdminBundle\Admin\AdminInterface;
-use Sonata\DoctrinePHPCRAdminBundle\Model\ModelManager;
-
-/**
- * A tree implementation to work with Doctrine PHPCR-ODM
- *
- * Your documents need to map all children with an Children mapping for the
- * tree to see its children. Not having the Children annotation is a
- * possibility to not show children you do not want to show.
- *
- * @author David Buchmann <david@liip.ch>
- * @author Uwe Jäger <uwej711@googlemail.com>
- */
-class PhpcrOdmTree implements TreeInterface
```

Comment:
according to travis there is a test for this class that now produces a fatal error when running the tests...